IronFoundry.Container.Test/ProcessRunnerTests.cs

```diff
@@ -193,6 +193,7 @@ public void WhenProcessFailsToStart_ThrowsException()
                     var si = CreateRunSpec("cmd.exe", new[] {"/C", "set"});
                     si.Credentials = user;
                     si.BufferedInputOutput = true;
+                    si.WorkingDirectory = @"c:\";
```

Comment:
Can you use `Environment.SystemDirectory` here instead?  Would be safer since the `C:` drive can be changed on Windows.